src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -579,7 +579,10 @@ class PartialReceivingPage extends Component {
     return apiClient.get(url)
       .then((response) => {
         this.setState({ values: {} }, () => {
-          this.setState({ values: parseResponse(response.data.data) });
+          this.setState({
+            values: parseResponse(response.data.data),
+            initialReceiptCandidates: parseResponse(response.data.data),
```

Comment:
why do we need to clear the values and then assign a value to this property?

Reply:
ask the one implementing it initially :see_no_evil: 
I didn't even see this clearing before, as I only cared about the place where I needed to pass the additional property.
Let's leave it as it is, because I don't know what potentially would be broken by that....

Reply:
:thinking:

Reply:
> why do we need to clear the values and then assign a value to this property? 

This is a really good question, this clearing values seems to be pointless here, and I don't know/remember any reason why it went in. I guess we could remove it and just check out if the page loads correctly.

> ask the one implementing it initially 🙈

:/